CMakeLists.txt

```diff
@@ -113,12 +114,12 @@ include(FetchContent)
 
 FetchContent_Declare(
   repo-core
-  GIT_REPOSITORY https://github.com/triton-inference-server/core.git
```

Comment:
Does this open us to any security issues?

Reply:
Resolved offline: The conclusion is that this doesn't pose a security risk.